.gitignore

```diff
@@ -0,0 +1 @@
+/test.md
```

Comment:
- your project it's missing node_modules.

_Check your project requirements and implement the requested installations.. could be consider to have installed linters from npm._

Reply:
Hello carlos-ssh, I have managed to fix what you've asked me to, but when I run the npx stylelint ., I am getting an error called unknown word CssSyntaxError, any ideas ?